docs/sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,39 @@
 Version history
 ===============
 
+5.8.1 (2018 March 22)
+---------------------
+
+File format fixes and improvements:
+
+* TIFF
+   - updated TiffWriter so that planes will no longer be split when using nonstandard
+     SamplesPerPixel e.g. images with 2 or 4 samples per pixel. This will ensure the ``TiffData``
+     elements represent the structure specified by the user. If users wish to split planes the 
+     ``ChannelSeparator`` and ``bfconvert`` provide the means to do this explicitly
+   - updated TiffWriter to use the correct logic for index checking when writing tiled images
+   - fixed a ``ClassCastException`` when the ``NEW_SUBFILE_TYPE`` tag has a non-standard type
+     or count such that the value is not inlined
+   - updated to also check the last IFD for an ImageJ comment in the scenario that the image has 
+     been processed by other software
+* NRRD (Nearly Raw Raster Data)
+   - added support for ``space directions`` and ``space units`` fields added in version 4
+* Evotec/PerkinElmer Opera Flex
+   - updated to read rather than calculate image offsets when a single tile is used
+
+Bug fixes and improvements:
+
+* limited the number of exceptions in the Bio-Formats plugins exporter when an unsupported pixel 
+  type is found
+* fake test images now allow for per-plane ExposureTime{X,Y,Z} and Position{X,Y,Z} keys in the INI file. 
```

Comment:
Really minor nitpick here, but can we avoid punctuation in this bullet point by just putting the 'For further details...' bit in brackets instead of starting a new sentence please.